--- ffmpeg.rb.orig	2023-09-23 00:09:10.000000000 +0200
+++ ffmpeg.rb	2023-09-23 21:06:30.000000000 +0200
@@ -51,18 +51,18 @@
   depends_on "librist"
   depends_on "libsoxr"
   depends_on "libvidstab"
-  depends_on "libvorbis"
+  #depends_on "libvorbis"
   depends_on "libvpx"
   depends_on "opencore-amr"
   depends_on "openjpeg"
-  depends_on "openssl@3"
+  depends_on "gnutls"
   depends_on "opus"
   depends_on "rubberband"
   depends_on "sdl2"
   depends_on "speex"
   depends_on "srt"
   depends_on "svt-av1"
-  depends_on "theora"
+  #depends_on "theora"
   depends_on "webp"
   depends_on "x264"
   depends_on "x265"
@@ -106,6 +106,11 @@
     sha256 "57e26caced5a1382cb639235f9555fc50e45e7bf8333f7c9ae3d49b3241d3f77"
   end
 
+  patch do
+    url "https://raw.githubusercontent.com/dafyk/mpv-macos-builds/main/ffmpeg.git-1231003c3c6d.patch.txt"
+    sha256 "cfb25204162b59dc03c0c506df33da2fa75945bc40159232388e8f11db98764f"
+  end
+
   def install
     args = %W[
       --cc=#{ENV.cc}
@@ -145,9 +150,7 @@
       --enable-libspeex
       --enable-libsrt
       --enable-libsvtav1
-      --enable-libtheora
       --enable-libvidstab
-      --enable-libvorbis
       --enable-libvpx
       --enable-libwebp
       --enable-libx264
@@ -156,7 +159,7 @@
       --enable-libxvid
       --enable-libzimg
       --enable-lzma
-      --enable-openssl
+      --enable-gnutls
 
       --disable-htmlpages
       --disable-podpages
