--- ffmpeg.rb.orig	2023-10-05 07:00:25.000000000 +0200
+++ ffmpeg.rb	2023-10-09 03:10:42.000000000 +0200
@@ -56,6 +56,7 @@
   depends_on "opencore-amr"
   depends_on "openjpeg"
   depends_on "openssl@3"
+  # depends_on "gnutls"
   depends_on "opus"
   depends_on "rubberband"
   depends_on "sdl2"
@@ -106,6 +107,11 @@
     sha256 "57e26caced5a1382cb639235f9555fc50e45e7bf8333f7c9ae3d49b3241d3f77"
   end
 
+  patch do
+    url "https://raw.githubusercontent.com/dafyk/mpv-macos-builds/main/ffmpeg.git-1231003c3c6d.patch.txt"
+    sha256 "cfb25204162b59dc03c0c506df33da2fa75945bc40159232388e8f11db98764f"
+  end
+
   def install
     ENV["MACOSX_DEPLOYMENT_TARGET"] = "10.13"
     # The new linker leads to duplicate symbol issue https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues/140
