--- mpv.rb.orig	2023-09-22 20:31:49.000000000 +0200
+++ mpv.rb	2023-09-23 02:44:31.000000000 +0200
@@ -20,7 +20,7 @@
   depends_on "libarchive"
   depends_on "libdvdnav"
   depends_on "little-cms2"
-  depends_on "luajit"
+  depends_on "lua@5.1"
   depends_on "mujs"
   depends_on "uchardet"
   depends_on "zimg"
@@ -32,8 +32,7 @@
 
   on_macos do
     depends_on "coreutils" => :recommended
-    depends_on "deus0ww/tap/dockutil@2" => :recommended if MacOS.version <  :big_sur
-    depends_on "deus0ww/tap/dockutil@3" => :recommended if MacOS.version >= :big_sur
+    depends_on "deus0ww/tap/dockutil@2" => :recommended
     depends_on "tag" => :recommended
     depends_on "trash" => :recommended
   end
@@ -62,7 +61,8 @@
       -Dlibmpv=true
       -Ddvdnav=enabled
 
-      --default-library=both
+      --default-library=static
+      --prefer-static
       --sysconfdir=#{pkgetc}
       --datadir=#{pkgshare}
       --mandir=#{man}
@@ -76,22 +76,11 @@
     system "meson", "compile", "-C", "build", "--verbose"
     system "meson", "install", "-C", "build"
 
-    if OS.mac?
-      # `pkg-config --libs mpv` includes libarchive, but that package is
-      # keg-only so it needs to look for the pkgconfig file in libarchive's opt
-      # path.
-      libarchive = Formula["libarchive"].opt_prefix
-      inreplace lib/"pkgconfig/mpv.pc" do |s|
-        s.gsub!(/^Requires\.private:(.*)\blibarchive\b(.*?)(,.*)?$/,
-                "Requires.private:\\1#{libarchive}/lib/pkgconfig/libarchive.pc\\3")
-      end
-    end
-
     bash_completion.install "etc/mpv.bash-completion" => "mpv"
     zsh_completion.install "etc/_mpv.zsh" => "_mpv"
 
     # Build, Fix, and Codesign App Bundle
-    system "python3.11", "TOOLS/osxbundle.py", "build/mpv", "--skip-deps"
+    system "python3.11", "TOOLS/osxbundle.py", "build/mpv"
     bindir = "build/mpv.app/Contents/MacOS/"
     rm   bindir + "mpv-bundle"
     mv   bindir + "mpv", bindir + "mpv-bundle"
