--- mpv.rb.orig	2023-09-22 20:31:49.000000000 +0200
+++ mpv.rb	2023-09-22 20:34:05.000000000 +0200
@@ -32,8 +32,7 @@
 
   on_macos do
     depends_on "coreutils" => :recommended
-    depends_on "deus0ww/tap/dockutil@2" => :recommended if MacOS.version <  :big_sur
-    depends_on "deus0ww/tap/dockutil@3" => :recommended if MacOS.version >= :big_sur
+    depends_on "deus0ww/tap/dockutil@2" => :recommended
     depends_on "tag" => :recommended
     depends_on "trash" => :recommended
   end
@@ -62,7 +61,8 @@
       -Dlibmpv=true
       -Ddvdnav=enabled
 
-      --default-library=both
+      --default-library=static
+      --prefer-static
       --sysconfdir=#{pkgetc}
       --datadir=#{pkgshare}
       --mandir=#{man}
