--- mpv/osdep/macos/swift_compat.swift.orig	2023-10-04 13:42:52.000000000 +0200
+++ mpv/osdep/macos/swift_compat.swift	2023-10-04 13:43:41.000000000 +0200
@@ -15,7 +15,7 @@
  * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#if !HAVE_MACOS_10_14_FEATURES
+#if false
 extension NSAppearance.Name {
     static let darkAqua: NSAppearance.Name = NSAppearance.Name(rawValue: "NSAppearanceNameDarkAqua")
     static let accessibilityHighContrastAqua: NSAppearance.Name = NSAppearance.Name(rawValue: "NSAppearanceNameAccessibilityAqua")
